File: L14.cpp
#include <iostream>
#include <vector>
using namespace std;
class Solution{
    public:
        string longestCommonPrefix(vector<string>& strs){
            if(strs.empty()){
                return "";
            }
            
            string store = strs[0]; 

            for(char i =0; i<strs.size(); i++){
                while(strs[i].find(store)!=0){
                    store = store.substr(0,)
                }
            }
        }
  
};

File: L14.java
class Solution{
    public String longestCommonPrefix(String[] strs){
        if(strs.length== 0){
            return "";
        }
        String store = strs[0];
        int count = store.length();
        for(char i=0;i<strs.length; i++){
            String Loop = strs[i];
            while(count > Loop.length() || !store.equals(Loop.substring(0,count))){
                count--;
            }
        }
    }
}